views/js/qtiCreator/tpl/toolbars/matchInteraction.score.tpl

```diff
@@ -1,5 +1,5 @@
 <input value="{{score}}" 
-       title="Score of this answer" 
+       title="{{__ 'The value of this field must be numeric'}}"
```

Comment:
This text duplicates the expected output of `validators.js`. I guess the tooltip gets its content from this property. Is there any way to make the tooltip use the message returned by the validator, to avoid the duplication?

Reply:
Yes, by simply removing the title from here, it is enough to have it on label and won't affect the tooltip text anymore.

Reply:
I was actually talking about the line you edited here: https://github.com/oat-sa/tao-core-ui-fe/pull/62/files?file-filters%5B%5D=.js#diff-10cc421c9f0d803e137dceb9a30c4407R51
`message: __('The value of this field must be numeric')`

I thought the message from the validator could be used in the interaction tooltip, since the text is identical. But I didn't check how difficult it would be to do that.

Reply:
ok, I see what you did now